docker/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM php:7.0
```

Comment:
does/would 7.2 work as well?

Reply:
Just tested it. Works flawlessly :-D
I even got additional "Namespace is not bound" warnings that were not reported with 7.0...